README.md

```diff
@@ -223,109 +223,37 @@ index.search(
 
 ### JSON <!-- omit in toc -->
 
-#### Basic JSON <!-- omit in toc -->
+#### Default JSON `GsonJsonHandler` <!-- omit in toc -->
 
-The default JSON can be created by calling the default constructor of `JsonbJsonHandler` class which will create a config of type `JsonbConfig` and using this config. It will initialize the mapper variable by calling the create method of `JsonbBuilder` class.
+The default JSON library used in the Meilisearch-Java SDK is `Gson`. You can easily use another library with the help of the ready-to-use `JsonHandler` Class.
 
-#### Creating a Custom `GsonJsonHandler` <!-- omit in toc -->
+*Notes*: We strongly recommend using the `Gson` library.
 
-To create a custom JSON handler, create an object of GsonJsonHandler and send the GSON object in the parameterized constructor.<br>
+#### Use a Custom `JacksonJsonHandler` <!-- omit in toc -->
 
-```java
-Gson gson = new GsonBuilder()
-             .disableHtmlEscaping()
-             .setFieldNamingPolicy(FieldNamingPolicy.UPPER_CAMEL_CASE)
-             .setPrettyPrinting()
-             .serializeNulls()
-             .create();
-private GsonJsonHandler jsonGson = new GsonJsonHandler(gson);
-jsonGson.encode("your_data");
-```
-
-#### Creating a Custom `JacksonJsonHandler` <!-- omit in toc -->
-
-Another method is to create an object of `JacksonJsonHandler` and set the required parameters. The supported option is an object of `ObjectMapper`. It's passed as a parameter to the `JacksonJsonHandler`’s parameterized constructor. This is used to initialize the mapper variable.
-
-The mapper variable is responsible for the encoding and decoding of the JSON.
-
-Using the custom JSON:
+Using the `jackson json` Library:
+Just create a `JacksonJsonHandler` object and send it through the `Config` object to the parameterized client.
```

Comment:
```suggestion
Create a `JacksonJsonHandler` object and send it through the `Config` object to the parameterized client.
```

`parameterized` ? Is this correct

> send it through the `Config` object

What do you mean? Initialize the config object with the handler?

Reply:
Yes!